frontend/src/static/css/style.scss

```diff
@@ -237,7 +237,20 @@ header{
             margin-top:0.1rem;
             font-size:1.0rem;
             color:mui-color("red");
-            font-weight:bold;
+
+            .loc{
+              font-weight:bolder;
+            }
+
+            span.fileType {
+                span::after{
+                  content: ', ';
+                }
+
+                span:last-child::after{
```

Comment:
let's follow the same as our coding stand for java, leave a space before the `{` left curly for consistency.

Reply:
OK, i've added the space for the portion i've written but the rest of the document does not seem to follow this coding standard

Reply:
Yes. This is because the automated checking tool that ongspxm has set up was not integrated with travis. So currently all the code quality needs to be manually reviewed. If you'd like, can try to restore it back up.